exp-player/addon/components/exp-overview/component.js

```diff
@@ -114,7 +114,12 @@ export default ExpFrameBaseComponent.extend(Validations, {
         var questions = this.get('questions');
         var responses = {};
         for (var i=0; i < questions.length; i++) {
-            responses[i] = questions[i].value;
+            if (i === 0) {
+                // Convert value to int bc select-input returns a string (e.g. "16" --> 16)
+                responses[i] = parseInt(questions[i].value);
```

Comment:
Given the time and date, I'll merge as is for now, but think back on this, and it can be changed in a followup PR if needed.